packages/core/admin/admin/src/pages/SettingsPage/index.js

```diff
@@ -43,13 +32,17 @@ function SettingsPage() {
   );
 
   // Creates the admin routes
-  const adminRoutes = useMemo(() => {
+  const adminRoutes = React.useMemo(() => {
     return makeUniqueRoutes(
       routes.map(({ to, Component, exact }) => createRoute(Component, to, exact))
     );
   }, [routes]);
 
-  const pluginsRoutes = createSectionsRoutes(settings);
+  const pluginsRoutes = Object.values(settings).flatMap((section) => {
+    const { links } = section;
+
+    return links.map((link) => createRoute(link.Component, link.to, link.exact || false));
+  });
```

Comment:
I would inline this to the return part and not use the `createRoute` utility, `createRoute` is a very lazy abstraction imo

Reply:
sure 🏁